bitcoind-rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/NetworkResult.scala

```diff
@@ -39,6 +39,7 @@ case class GetNetworkInfoResult(
     subversion: String,
     protocolversion: Int,
     localservices: String,
+    localservicesnames: Option[Vector[String]],
```

Comment:
I believe we have a type for this, also could you make sure that there is an assert on this field in the test for this RPC?